docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -66,7 +66,7 @@ ifndef::community[]
 //* You have a subscription to {product-long-kafka}. For more information about signing up, see *<@SME: Where to link?>*.
 endif::[]
 * https://adoptopenjdk.net/[JDK^] 11 or later is installed.
-* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
+* Latest LTS version of OpenJDK is recommended 
```

Comment:
@wtrocki So sorry for the delay! I somehow missed the reviewer ping. Apologies!

Are you meaning this to be a separate prereq? It seems like an add-on recommendation to the JDK 11 prereq.

So this?

```
* https://adoptopenjdk.net/[JDK^] 11 or later is installed. (Latest LTS version of OpenJDK is recommended.)
```

Or this?

```
* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
* Latest LTS version of OpenJDK is installed (recommended).
```

Reply:
I do not mind either way. Key is to drop Oracle 🙈  JDK and we have tested latest LTS (17 on windows). 
First option is good IMHO

Reply:
@wtrocki Okay, sounds good. If you can apply the first option to all of your cases in the PR, I can merge. Thanks.